src/style.css

```diff
@@ -0,0 +1,16 @@
+html,
```

Comment:
Maybe combine this with createMuiTheme in index.js?

Reply:
tried that, did not go too well, or I did not figure out the syntax